tests/recipes/test_full_finetune_single_device.py

```diff
@@ -51,15 +51,16 @@ def _fetch_expected_loss_values(self):
         return [10.5074, 10.5563, 10.5152, 10.4851]
 
     @pytest.mark.integration_test
-    def test_loss(self, tmpdir, monkeypatch):
+    @pytest.mark.parametrize("config", ["full_single_device_low_memory"])
```

Comment:
Should still be running on the original recipe too, no?

Reply:
ah yes, good catch!